policies/templates/gcp_gke_cluster_version_v1.yaml

```
@@ -0,0 +1,118 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-cluster-version-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GKEClusterVersionConstraintV1
+      validation:
+        openAPIV3Schema:
+          properties:
+            mode:
+              type: string
+              enum: [denylist, allowlist]
+              description: "String identifying the operational mode, allowlist or denylist. In allowlist mode,
+              GKE clusters are only allowed to have a master version specified in the 'master_versions' parameter. In denylist mode,
+              resources are allowed any master versions except those listed in the 'master_versions' parameter."
+            version_type:
+              type: string
+              enum: [currentMasterVersion, currentNodeVersion]
```

These values seem unnecessarily long/confusing.

```suggestion
              enum: [master, node]
```

---

These values are the actual field names in CAI and are used to find those properties. Should they still be changed?

---

Ok- have changed to "master" and "node" as the values. Let me know your thoughts- thanks!

---

We shouldn't expect users to know the CAI values, so thanks for updating.